### PR TITLE
Add info on how to access the raw id_token

### PR DESCRIPTION
**Pull request by luisrudge**

### CODE REVIEW

File: articles/libraries/auth0-spa-js/index.md [hunk 1]
```diff
@@ -218,36 +218,39 @@ $('#getToken_audience').click(async () => {
...
 });
 ```
 
 ### Get ID Token Claims
 
 ```js
-$('#getIdTokenClaims').click(async () => {
+$("#getIdTokenClaims").click(async () => {
   const claims = await auth0.getIdTokenClaims();
+  // if you need the raw id_token, you can access it
```
Comment (lbalmaceda) [inline articles/libraries/auth0-spa-js/index.md:231]: Please remove the rest of the changes and only keep this. It seems your code formatting tool is refactoring everything else.

**Review by lbalmaceda — CHANGES_REQUESTED** [review]

**Comment by lbalmaceda** [comment]
@luisrudge see my comment above. Build is broken.

File: articles/libraries/auth0-spa-js/index.md (hunk 1 above)
Comment (luisrudge) [reply articles/libraries/auth0-spa-js/index.md:231]: fixed in https://github.com/auth0/docs/pull/8345

**Comment by luisrudge** [comment]
fixed in https://github.com/auth0/docs/pull/8345

**Pull request closed without merging**